onnx/external_data_helper.py

```diff
@@ -128,6 +129,8 @@ def convert_model_to_external_data(
     if all_tensors_to_one_file:
         file_name = str(uuid.uuid1())
         if location:
+            if os.path.isabs(location):
```

Comment:
Should we allow absolute path relative to the model path?

Reply:
I think not. Absolute path will leads to problems like "file not found" if we move models around, which should be common.

Reply:
And, path `A` can always be a relative to path `B` on a given system.